app/controllers/friendships_controller.rb

```diff
@@ -4,34 +4,36 @@ def new
   end
 
   def create
-
     if params[:friend_information][:sender] != params[:friend_information][:receiver]
-
-    @new_friendship = Friendship.new(sender:params[:friend_information][:sender], 
-                                     receiver: params[:friend_information][:receiver], status: 0)
-    if @new_friendship.save 
-      redirect_to request.referrer || root_url
-    end
-
+      @new_friendship = Friendship.new(sender:params[:friend_information][:sender], 
+                                       receiver: params[:friend_information][:receiver], status: 0)
+      if @new_friendship.save 
+        redirect_to request.referrer || root_url
+      end
     else
-
-        flash[:danger] = 'You cant send an invitation to yourself!'
+      flash[:danger] = 'You cant send an invitation to yourself!'
     end
   end
 
   def edit
     @fs = Friendship.find_by(id: params[:id])
     @fs.status = !@fs.status
-    if @fs.save 
-      redirect_to request.referrer || root_url
+    if @fs.save
+      @fs_inverse = Friendship.new(sender: @fs.receiver, receiver: @fs.sender, status: true)
+      if @fs_inverse.save
+        redirect_to request.referrer || root_url
+      end
     end
   end
```

Comment:
[OPTIONAL]
Why are you doing this in `edit` action instead of `update` action? [Reference](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions)

Also, consider wrapping those 2 updates in a transaction so that if one fails (for any reason), the change is rolled back.